io.gpl.wxmaxima.yml

```
@@ -0,0 +1,76 @@
+app-id: io.github.wxmaxima_developers.wxmaxima
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host:rw
+      
+modules:
+  - name: gnuplot
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
+      - --with-qt=no
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/gnuplot/5.4.10/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: sbcl
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - cp -r ./sbcl/* /app/
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/f/sbcl-2.3.9.tar.gz
+        sha256: 4b685a3b834d1faf0976e936750062e4c8ad5718daf115075615402913b69121
+  - name: maxima
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
+      - --enable-sbcl
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/maxima/Maxima-source/5.47.0-source/maxima-5.47.0.tar.gz
+        sha256: 9104021b24fd53e8c03a983509cb42e937a925e8c0c85c335d7709a14fd40f7a
+  - name: wxwidgets
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --prefix=/app/
+      - --with-gtk
+      - --enable-webviewwebkit
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.3/wxWidgets-3.2.3.tar.bz2
+        sha256: c170ab67c7e167387162276aea84e055ee58424486404bba692c401730d1a67a
+  - name: wxmaxima
+    buildsystem: cmake-ninja
+    no-autogen: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app/
+      - -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.2 
+      - -DwxWidgets_LIBRARIES=app/lib
```

```suggestion
      - -DwxWidgets_LIBRARIES=/app/lib
```

I'm pretty sure you are missing the `/`